pallets/parachain-staking/src/mock.rs

```diff
@@ -115,6 +115,7 @@ impl crate::pallet::Config for Test {
 	type MaxCollatorsPerNominator = MaxCollatorsPerNominator;
 	type MaxFee = MaxFee;
 	type MinCollatorStk = MinCollatorStk;
+	type MinCollatorCandidateStk = MinCollatorStk;
```

Comment:
If we want, we can set this to be a different value from `MinCollatorStk`. The new constant is the minimum amount reserved for an account to become a collator candidate. The `MinCollatorStk` is now the minimum required for a collator candidate to be chosen for the round in `SelectedCandidates` (which is the set of collators for the round).

Why? If we want to lower the capital requirements for becoming a collator candidate such that accounts can become a candidate with less funds, but they still need to get a certain backing in order to be selected in `SelectedCandidates`. This was mentioned to me by @artkaseman

Reply:
To me this makes sense in the opposite case of what you described: We want to _raise_ the cost of joining the candidates (because there are too many) but we don't want to boot anyone from the active set (yet).

The case where we lower the cost to become a candidate seems like it might have unintended consequences. Imagine we lower the cost to become a candidate such that it is lower than the total stake to enter the active set. I can image a lot of stakers saying "wth, I made the minimum stake requirement, I got nominators, and I'm in the top N. Why am I not active?"

That being said, if people want it I'm fine with it.

Reply:
> I can image a lot of stakers saying "wth, I made the minimum stake requirement, I got nominators, and I'm in the top N. Why am I not active?"

If we monitor the chain and make sure that there are at least `TotalSelectedCandidates` collator candidates with the `MinCollatorStk`, then this will never happen because anyone in the top `N` will be put in `SelectedCandidates` for the round..